doc/source/whatsnew/v0.25.0.rst

```diff
@@ -608,6 +608,7 @@ MultiIndex
 ^^^^^^^^^^
 
 - Bug in which incorrect exception raised by :class:`Timedelta` when testing the membership of :class:`MultiIndex` (:issue:`24570`)
+- :meth:`MultiIndex.isin` now raises ``ValueError`` when args are too short (:issue:`26622`)
 -
```

Comment:
this should probably be a bit clearer as well.. something like..

...when items in values are not the same length as the number of levels in the MultiIndex...

Reply:
done